src/Esfa.Vacancy.Register.Api/DependencyResolution/DefaultRegistry.cs

```diff
@@ -15,22 +15,23 @@
 // </copyright>
 // --------------------------------------------------------------------------------------------------------------------
 
+using MediatR;
+
 namespace Esfa.Vacancy.Register.Api.DependencyResolution
 {
     public class DefaultRegistry : StructureMap.Registry
     {
-        #region Constructors and Destructors
+        private const string ServiceName = "Esfa.Vacancy.Register";
 
         public DefaultRegistry()
         {
             Scan(
                 scan => {
-                    scan.TheCallingAssembly();
-                    scan.WithDefaultConventions();
+                    scan.AssembliesFromApplicationBaseDirectory(a => a.GetName().Name.StartsWith(ServiceName));
```

Comment:
Does this mean that we don't have to explicitly register each and every Repo etc... is it taken care by convention?